pkg/target/host.go

```diff
@@ -120,13 +118,15 @@ func (h *Host) Stream(wg *sync.WaitGroup, updateHostHealthStatus chan *HostStatu
 }
 
 func (h *Host) closeUpdateHostConnection(updateHostHealthStatus chan *HostStatus) {
+	// TODO (grzkv): This is called in case connection is already broken.
+	// what is the guarantee that it will not get stuck?
```

Comment:
Not sure if I understand the question fully!

Reply:
If a connection is broken the guarantees on how closing it will behave are not strong. Is it safe to close already broken connection?

Reply:
@gksinghjsr ^^^?